app/routes/photos/components/GalleryPictureModal.css

```diff
@@ -2,7 +2,7 @@
 
 .content {
   padding: 0;
-  max-width: 1100px;
+  width: 1100px;
```

Comment:
Should ideally use `var(--lego-max-width)` (which is equal to `1100px`), but oh well 🤷🏼